<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>812</width>
    <height>570</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Source Code Variable</family>
    <pointsize>15</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 38, 38);
font: 15pt &quot;Source Code Variable&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="endbutton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Variable</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 143, 197);
font: 28pt;
color: rgb(240, 241, 250);</string>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_preview">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>390</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;
background-color: rgb(46, 86, 105);
color: rgb(240, 241, 250);</string>
    </property>
    <property name="text">
     <string>Toggle Preview</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gamemode">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>510</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;
background-color: rgb(143, 190, 212);
color: #2e5669;</string>
    </property>
    <property name="text">
     <string>Go Game!</string>
    </property>
   </widget>
   <widget class="QLabel" name="preview_camera">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Variable</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8fbed4;</string>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="time_disp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Variable</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8fbed4;</string>
    </property>
    <property name="text">
     <string>Yaw</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Variable</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8fbed4;</string>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Variable</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8fbed4;</string>
    </property>
    <property name="text">
     <string>Pitch</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="yaw_disp">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>550</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="roll_disp">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>550</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="pitch_disp">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>580</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Variable</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8fbed4;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="z_disp">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>580</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>580</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Variable</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8fbed4;</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="y_disp">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>580</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="x_disp">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>580</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>580</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Variable</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8fbed4;</string>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="config_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;
background-color: rgb(46, 86, 105);
color: rgb(240, 241, 250);</string>
    </property>
    <property name="text">
     <string>Config</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Variable</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 143, 197);
font: 28pt;
color: rgb(240, 241, 250);</string>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="fps_disp">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>580</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>550</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Variable</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8fbed4;</string>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="centerButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>220</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(49, 143, 197);
font: 15pt;
color: rgb(240, 241, 250);</string>
    </property>
    <property name="text">
     <string>Center</string>
    </property>
   </widget>
   <widget class="QPushButton" name="always_on_top">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;
background-color: rgb(143, 190, 212);
color: #2e5669;</string>
    </property>
    <property name="text">
     <string>Always On Top</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>570</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt;
background-color: rgb(85, 0, 0);
color: rgb(240, 241, 250);</string>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
